lms/djangoapps/lti_provider/views.py

```diff
@@ -189,13 +189,11 @@ def render_courseware(request, lti_params):
         'disable_accordion': True,
         'allow_iframing': True,
         'disable_header': True,
```

Comment:
Perhaps not for this PR, but it would be great if the header were in a block as well, so that the chromeless template could just override that block to be empty, rather than requiring the caller to pass in this variable.